cache-config/t3c-apply/torequest/torequest.go

```diff
@@ -1169,17 +1175,25 @@ func (r *TrafficOpsReq) UpdateTrafficOps(syncdsUpdate *UpdateStatus) error {
 
 	// TODO: The boolean flags/representation can be removed after ATC (v7.0+)
 	if !r.Cfg.ReportOnly && !r.Cfg.NoUnsetUpdateFlag {
+		start := time.Now()
+		apply := []string{}
+		var b bool
```

Comment:
Another nitpick: the scope of this variable is a bit too long for a single character. Variable name lengths should correspond to their scope. 

Single-letter vars are ok for just a couple lines, but for a dozen or so like here, something more descriptive like "updateFlagVal" would be easier to read.